pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -121,21 +128,23 @@ void init_circuit_add_classical_op(
              const std::string &wasm_uid,
              const std::vector<unsigned> &i32list_i,
              const std::vector<unsigned> &i32list_o,
-             const std::vector<Bit> &args) -> Circuit & {
+             const std::vector<Bit> &args,
+             const py::kwargs &kwargs) -> Circuit * {
             unsigned n_args = args.size();
             std::shared_ptr<WASMOp> op = std::make_shared<WASMOp>(
                 n_args, i32list_i, i32list_o, funcname, wasm_uid);
-            circ.add_op(op, args);
-            return circ;
+
+            return add_gate_method<Bit>(&circ, op, args, kwargs);
           },
-          "See :py:meth:`add_wasm`.", py::arg("funcname"), py::arg("wasm_uid"),
+          "See :py:meth:`_add_wasm`.", py::arg("funcname"), py::arg("wasm_uid"),
```

Comment:
I have changes that, but might be the right things to remove the docs here anyway because it is not shown in the docs anyway for all _add_wasm functions.